sentry/src/db/spendable.rs

```diff
@@ -134,6 +137,7 @@ mod test {
     #[tokio::test]
     async fn it_inserts_and_fetches_and_updates_spendable() {
         let database = DATABASE_POOL.get().await.expect("Should get a DB pool");
+        let app = setup_dummy_app().await;
 
         setup_test_migrations(database.pool.clone())
             .await
```

Comment:
When using the `setup_dummy_app` it does a lot more work and also runs the `setup_test_migrations`.
I suggest to just use the config directly instead of setting up a whole app: `GANACHE_CONFIG`